controllers/profile_controller.js

```diff
@@ -31,8 +32,8 @@ const controller = {
 
   //Method GET: to Show form to edit profile:
   async editProfile(req, res) {
-    const currentUserId = req.params.user_id;
-    res.render("./profiles/profile_edit.ejs", { currentUserId });
+    const currentUser = await Users.findById(req.params.user_id);
```

Comment:
Above you use `profileOwner`. here you use `currentUser`. Remember rule #1: consistency.

Choose one and stick with it.

---

I highly recommend you to *NOT* use `currentUser` since that is what it set for authenticated user `session.currentUser` and will cause confusion